draft-zip.md

```diff
@@ -0,0 +1,107 @@
+```
+ZIP: 
+Title: Deposit 60% of transaction fees to the Zcash Sustainability Fund
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits:
+Status: Draft
+Category: Ecosystem
+Created: 2023-09-21
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”, and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+The term “network upgrade” in this document is to be interpreted as described in ZIP 200. [2]
+
+The term “Block Rewards” in this document is to be interpreted as described in ZIP TBD [3]
+
+The term “Issuance” in this document is to be interpreted as described in ZIP TBD [3]
+
+“We” - the ZIP authors, owners listed in the above front matter
+
+“`ZSF_BALANCE_AFTER[h]`” is balance of the Zcash Sustainability Fund as defined in ZIP ###
+
+# Abstract
+
+This ZIP proposes a modification to transaction fees, diverting 60% of transaction fees back into the `ZSF_BALANCE_AFTER[h]`, while  the destination of the remaining 40% is unchanged and goes to the block miner. This proposal effectively “unmints” a portion of transaction fees, contributing to a deflationary effect and offering long-term support for the Zcash network.
+
+This ZIP attempts to establish a symbiotic relationship between miner incentives and sustained network growth. It achieves this by splitting transaction fees: 40% goes directly to miners, incentivizing them to include transactions, while the remaining 60% is deposited into the  `ZSF_BALANCE_AFTER[h]`. This approach mitigates a "bootstrapping problem", a problem that arises when 100% of transaction fees go to the ZSF and miners are not incentivized to include transactions in block. This ZIP navigates this problem by ensuring miners continue to receive direct rewards for including transactions, while still contributing to the ZSF.
+
+Implementing this change allows the ZSF to accrue value earlier. By ensuring a consistent source of funding, the ZSF contributes to bolstering the Zcash network’s long-term security and sustainability.
+
+# Motivation
+
+While ZIP-XXX (Establishing the Zcash Sustainability Fund) describes a method by which funds can be added to the Zcash Sustainability Fund by a voluntary `ZSF_DEPOSIT` transaction field. The default value of this field is zero and it is left up to the app, wallet, and mining software implementers to make use of it.
+
+This ZIP takes a much more explicit and non-optional approach, mandating at the protocol level that 60% of transaction fees be deposited into the ZSF. As noted above, implementing this change allows the ZSF to accrue value earlier and contribute to future network sustainability.
```

Comment:
The ZIP Editors have a concern that it is not obvious to us, or to other community participants trying to assess the consequences of this ZIP, how much money is actually involved here. The most relevant information would be to estimate the amount of fees per block that are likely to be paid, based on historical transaction usage (but excluding "sandblasting" transactions), under the ZIP 317 fee regime.

Reply:
I've added the [Estimated impact on miners](https://github.com/eigerco/zips/blob/ZSF-50-50/draft-zip.md#estimated-impact-on-miners) section that does just that.

I've used a [simple tool](https://github.com/eigerco/zsf-fee-estimator) that connects to a Zcash node and extracts statistics for the last 100k blocks. The tool simplifies the logical action count calculation compared to ZIP-317 by not using the exact serialized sizes for transparent inputs and outputs and rather just using the number of those inputs and outputs.

This allowed me to simplify my code and should not affect the estimation much, as ZIP-317 claims that the parameters were chosen to represent typical inputs and outputs. Even if not the impact should be small and should not affect the discussion in the context of this ZIP.

Let me know if you think this is not an acceptable simplification.